public/css/main.css

```diff
@@ -66,46 +66,3 @@ header {
 .next-question {
     margin-top: 42px;
 }
-
-.hide-with-space {
```

Comment:
Did you delete the .hide-with-space and .pic-table?

Reply:
Yeah I accidentally did - thanks for catching that!